src/libs/OptionsListUtils.js

```diff
@@ -412,6 +413,9 @@ function getLastMessageTextForReport(report) {
         lastActionName === CONST.REPORT.ACTIONS.TYPE.TASKCANCELLED
     ) {
         lastMessageTextFromReport = lodashGet(lastReportAction, 'message[0].text', '');
+    } else if (lastChildType === CONST.REPORT.TYPE.TASK) {
+        // This is because for task report action the server sends for e.g '<' while the task title is '<<<test' in the lastMessageText
+        lastMessageTextFromReport = report ? report.lastMessageHtml || '' : '';
```

Comment:
Ah I see, I didn't see that this line has `lastMessageHtml` instead of `lastMessageText`. I'd say lets not add it. We should fix the root cause not the symptom, and as we all agree this is root cause in the backend 🙂